modules/programs/ledger.nix

```diff
@@ -0,0 +1,196 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.programs.ledger;
+  package = pkgs.ledger;
+  boolOption = value: name: if value == true then "--${name}" else "";
```

Comment:
you could use `optionalString` to simplify:

```nix
let
  boolOption = value: name: optionalString value "--${name}";
in {}
```

The `== true` seems redundant.